sklearn/ensemble/_forest.py

```diff
@@ -1177,6 +1177,7 @@ class RandomForestClassifier(ForestClassifier):
     A random forest is a meta estimator that fits a number of decision tree
     classifiers on various sub-samples of the dataset and uses averaging to
     improve the predictive accuracy and control over-fitting.
+    Trees in a forest use the parameter `splitter="best"`.
```

Comment:
I would rephrase a bit to not mention the parameter first since this is not in the list of parameter.
```suggestion
    Trees in the forest use the best split strategy, i.e. equivalent to passing `splitter="best"`
    to the underlying :class:`~sklearn.tree.DecisionTreeClassifier`.
```